procrastinate/contrib/django/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class ProcrastinateConfig(AppConfig):
+    name = "procrastinate.contrib.django"
+    label = "procrastinate_django"
```

Comment:
I think "procrastinate" could be enough in this case. Also, this would make the verbose name (which defaults on `label.title()`) cleaner.

Note: I'm not 100% sure.